README.md

```
@@ -126,13 +137,13 @@ const uppercaseString = zagora()
 	})
 	.handlerSync((input, err) => {
 		if (input === "network") {
-			return err.network({
+			throw err.network({
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)